coralproject/talk
# [CORL-2424]: Add rel=ugc to comment links

## What does this PR do?

These changes add `rel="ugc"` to links that users add to comments. [This page](https://moz.com/blog/nofollow-sponsored-ugc) includes more context, but basically, `ugc` stands for user-generated content, and it provides a hint to search engines about the link. This is for https://vmproduct.atlassian.net/browse/CORL-2424.

## What changes to the GraphQL/Database Schema does this PR introduce?

None

## Does this PR introduce any new environment variables or feature flags? 

No

## How do I test this PR?

Paste or type links into a comment. See that when you submit the link and it's posted, if you inspect it in dev tools, it has `ugc` included within its `rel` attribute. 
 
## How do we deploy this PR?
